test/Microsoft.Dnx.Testing.CommonUtils/ProjectResolver/IProjectResolver.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+
+using System.Collections.Generic;
```

Comment:
@davidfowl The only issue with internalizing Project resolver like this is that in case these files changes inside Tooling in the future, we'll need to duplicate it here as well. This may become a headache later?